src/test/java/com/epam/tc/hw9/PositiveApiTestsRestTrelloService.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.tc.hw9;
+
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class PositiveApiTestsRestTrelloService extends BaseTest {
+    @DataProvider
+    public Object[][] boardNameData() {
+        return new Object[][]{
+                {"FirstBoard"},
+                {"SecondBoard"},
+                {"ThirdBoard"},
+        };
+    }
+
+    @Test(description = "A test that verifies the correctness of creating a board",
+            dataProvider = "boardNameData")
+    public void createBoardTest(String nameBoard) {
+        restTrelloService.createNewBoard(nameBoard)
+                .then()
+                .statusCode(SC_OK);
+        boards = restTrelloService.getAllBoards();
+        Assertions.assertThat(boards.length).isEqualTo(defaultCountBoard).as("Count boards not equal expected");
```

Comment:
pls, add separate class for all assertions

Reply:
+